init-keys.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+set -e
+
+
+SECRET_VOLUME="beekeeper-config_bk-secrets"
+
+
+
+if [ "${1}_" == "_" ] ; then
+    echo "./init-keys.sh [new|test]"
+    echo "     new: create new CA and server keys"
+    echo "     test: use existing test CA and server keys (WARNING: only for testing/development"
+    exit 1
+fi
+
+
+
+set +e
+docker volume inspect ${SECRET_VOLUME}
+if [ $? -eq 0 ] ; then
+    echo "Error: Docker volume ${SECRET_VOLUME} already exists. Please delete first."
+    exit 1
+fi
+set -e
+
+
+if [ "${1}_" == "new_" ] ; then    
+
+    set -x
+    pushd bk-config
+    docker build -t sagecontinuum/bk-config .
+    popd
+    docker run -ti --rm --name bk-config -v ${SECRET_VOLUME}:/usr/lib/sage/ sagecontinuum/bk-config init-keys.sh
+
+    set +x
+    exit 0
+fi
+
+if [ "${1}_" == "test_" ] ; then
```

Comment:
Can we call this as part of a Workflow CI unit-test?  would there be value in that?

Reply:
done